### PR TITLE
Change openssl configuration in ci

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: .github/workflows/rotki_ci.yml [hunk 1]
```diff
@@ -139,28 +139,6 @@ jobs:
...
 
-  lint-backend:
-    name: 'Backend lint'
-    needs: ['check-changes']
-    if: ${{ needs.check-changes.outputs.backend_tasks }}
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Setup python
```
Comment (kelsos) [inline .github/workflows/rotki_ci.yml:150]: Why did you remove the lint task?

Comment (yabirgb) [reply .github/workflows/rotki_ci.yml:150]: Just to speed the process while I test if this works.

**Review by kelsos — APPROVED** [review]
lgtm

**Comment by kelsos** [comment]
@yabirgb btw if you want to test configuration changes only and run all the tests you can add `[run all]` in the commit message.

**Comment by yabirgb** [comment]
> @yabirgb btw if you want to test configuration changes only and run all the tests you can add `[run all]` in the commit message.

A nice. I didn't know it skipped the rule of the file changed in folders
